web-marketplace/src/components/organisms/ListProject/ListProject.tsx

```diff
@@ -31,8 +32,9 @@ const ListProject = () => {
     walletsUiConfig,
     onButtonClick,
   } = useLoginData({ createProject: true, isConnectingRef });
-  const { isIssuer } = useProfileItems({});
-
+  const { isIssuer, isLoadingIsIssuer } = useQueryIsIssuer({
+    address: activeAccount?.addr,
+  });
```

Comment:
in `useProfileItems` we do other queries which are not needed for this particular use case where we just need `isIssuer`
since it was part of the main nav, it was affecting the app performance generally